src/NUnitFramework/framework/Assert.cs

```diff
@@ -59,6 +59,7 @@ public partial class Assert
         /// </summary>
         /// <param name="a"></param>
         /// <param name="b"></param>
+        /// <exception cref="InvalidOperationException">Assert.Equals should not be used for Assertions</exception>
```

Comment:
Seems unnecessary, the exception is always thrown because there is no reason to call this and it looks similar to Assert.AreEqual.